main.rb

```diff
@@ -0,0 +1,80 @@
+require 'terminal-table/import'
+require_relative 'constant'
+require_relative 'validator'
+require_relative 'grocery_item'
+
+
+# prompt: Add items to the cart
+puts "Please enter all the items purchased separated by a comma"
+
+inputs = gets.chomp
+
+# Converting items string to an array
+items = inputs.split(',')
+
+# remove unwanted spaces in the items
+items.map! { |item| item.strip }
+
+# valid-input validation
+unless Validation::validateInput(items)
+    exit
+end
+
+# mapping over items and converting the array of strings to symbols
+items.map! { |item| Constants::GROCERY_ITEMS_MAPPINGS[item.to_sym] }
```

Comment:
We can add some logic to filter out inputs that are not part of the `GROCERY_ITEMS_MAPPINGS` keys

Reply:
I have tried to put that logic in the main.rb file:

- Created a new instance of the class Cart
- looped over items, while assigning the Constants to a variable
- then called the "add_items" method on cart with the (grocery_item) param which was assigned to the Constants.

main.rb:
https://github.com/chandnissirohi/Ruby-grocery-store-2/commit/6dcd9a39405db333e1755ca50a0ffa05797f1908